Userland/Libraries/LibPDF/Fonts/PDFFont.cpp

```diff
@@ -14,7 +15,7 @@
 
 namespace PDF {
 
-static bool is_standard_latin_font(DeprecatedFlyString const& font)
+[[maybe_unused]] static bool is_standard_latin_font(DeprecatedFlyString const& font)
```

Comment:
what's this for?

Reply:
This *was* used to determine if a font was a standard font, which in turn determined whether we should look for a replacement font in the system. This PR changes the latter bit so now a replacement font is looked for whenever a font is not embedded in the PDF, which covers more cases. I felt like this function would still be useful at some point though, which is why I kept it -- but it's in git anyway, so I can remove it instead if preferred.